NuRadioReco/detector/detector_base.py

```diff
@@ -974,6 +974,32 @@ def get_channel_group_id(self, station_id, channel_id):
             return channel_id
         else:
             return res['channel_group_id']
+        
+    def get_antenna_mode(self, station_id, channel_id):
+        """
+        returns the antenna mode of a given channel - this is specific to LOFAR antennae, as they operate in either inner or outer mode.
```

Comment:
The plural of a radio antenna is antennas. Antennae is only for those from insects (SCNR).